ts/frames.ts

```diff
@@ -58,6 +58,11 @@ export const frameTypes = [
     'VentilatedBoxcarCC',
     'WaterCar',
     'waycar',
+    'Class125',
+    'Class125_Tender',
+    'SideDumpCar',
+    'DRGW_Gondola_33FT',
+    'dspp_baggage',
```

Comment:
I'd like to keep this list sorted so it's easier to find a specific entry.

Reply:
sorted how ?